libs/auth/env.go

```diff
@@ -38,3 +38,17 @@ func GetEnvFor(name string) (string, bool) {
 
 	return "", false
 }
+
+func EnvVars() []string {
```

Comment:
Please include a description explaining why this function exists inline.

I was puzzled and had to go back to the PR description to figure it out.